mne/epochs.py

```diff
@@ -178,6 +178,11 @@ def __init__(self, info, data, events, event_id, tmin, tmax,
         del event_id
 
         if events is not None:  # RtEpochs can have events=None
+
+            if not np.array_equal(events, np.array(events, dtype=int)):
```

Comment:
but is the users passes a float array that only contains integers, we don't want to throw an error do we?